backend/tests/routes/test_files.py

```diff
@@ -5,11 +5,14 @@
 
 from api.constants import constants
 from api.database import get_local_fpath_for
+from api.routes.files import task_queue
 
 
 def test_upload_file_correct_data(
-    logged_in_client, project_id, test_file, test_file_hash
+    logged_in_client, project_id, test_file, test_file_hash, mocker
 ):
+    task_queue_mock = mocker.patch.object(task_queue, "enqueue")
+    task_queue_mock.return_value = True
```

Comment:
I'm not quite sure how to handle tests related to S3 uploads, here I hacked the S3 upload part.